README.md

```diff
@@ -163,6 +166,190 @@ Then the page should be accessible checking only: document-title, label
 Then the page should be accessible according to: best-practice and checking: aria-roles, definition-list
 ```
 
+# RSpec
+
+## Configuration
+
+1. Require rspec matchers: in `spec/spec_helper.rb`, `spec/support/env.rb` or similar.
```

Comment:
Let's drop the `spec/support/env.rb` since you already have the conventional `spec/spec_helper.rb`